CHANGELOG.md

```diff
@@ -24,6 +24,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - JSON API: `fundchannel_cancel` is extended to work before funding broadcast.
 - JSON API: The parameter `exclude` of `getroute` now also support node-id.
 - JSON-API: `pay` can exclude error nodes if the failcode of `sendpay` has the NODE bit set
+- JSON-API: `fundchannel` now uses `amount` as the parameter name to replace `satoshi`
+- JSON-API: `fundchannel_start` now uses `amount` as the parameter name to replace `satoshi`
```

Comment:
Maybe move these lines under `## Deprecated`?  I scan those so I know what to remove in future releases...